tests/ha/yast_cluster_init.pm

```diff
@@ -68,17 +77,28 @@ sub run {
     wait_screen_change { send_key 'alt-n' };
 
     # Cluster service settings screen
+    # This screen is different depending on the yast2-cluster version.
+    # In 15-SP2 Alt-E is required to enable the cluster at boot time,
+    # and Alt-a to start cluster now. In older versions, Alt-a is
+    # required to start the cluster on boot and Alt-S to start the
+    # cluster now. The following combination of send_key calls tries
+    # to cover most cases
     assert_screen 'yast-cluster-service';
-    send_key 'alt-a';    # Start pacemaker during boot
+    send_key 'alt-E';    # Enable cluster
+    wait_still_screen;
     send_key 'alt-S';    # Start pacemaker now
-    wait_still_screen 5;
+    wait_still_screen;
+    send_key 'alt-a';    # Start pacemaker during boot
+    wait_still_screen;
+    save_screenshot;
     wait_screen_change { send_key "alt-n" };
     save_screenshot;
     wait_still_screen 5;
-    save_screenshot;
+    wait_serial('yast2-cluster-status-0', 90) || die "'yast2 cluster' didn't finish";
     save_state;
 
     # Generate ssh key
+    type_string "test -f /root/.ssh/id_rsa && rm -rf /root/.ssh\n";
```

Comment:
```suggestion
    assert_script_run "rm -rf /root/.ssh";
```

With `-f` the `rm` command always return 0, so no previous `test` needed.

Reply:
@alvarocarvajald  Is the idea not to remove the existing directory if there is no private key but keep anything else? If not I agree with @ricardobranco777. Otherwise your test is fine.

Reply:
Actually, the `test -f` is not there to prevent the `rm` from failing, but to only execute the `rm` command if needed. But you're correct in that if the only action is to remove `/root/.ssh`, we can remove it always.

However, IMO is a bad practice to do `assert_script_run "rm -rf ..."` as - as you say - the `-f` flag makes `rm` return 0 always, so you are not expecting the asserted command to fail.

I'll drop the `test -f` part of the command. Will push change and verification runs shortly.

Additionally, as a matter of fact, we could drop L101 altogether (it was not the goal of this PR TBH). It shouldn't be needed. This is only there due to the changes introduced in #10288 and then fixed for SLES4SAP and HA in #10339 and #10349, but not fixed for `tests/ha/yast_cluster_init` as apparently this module is currently not being scheduled to run. This will be solved by #10378.

Reply:
Verification run:

http://mango.suse.de/tests/2879
http://mango.suse.de/tests/2880
http://mango.suse.de/tests/2878